### DISCUSSION TITLE
[HW2] Trouble understanding the baseline.

### DISCUSSION THREAD

**Discussion by ggoggam** [seominjoon/kaist-ai605#33] [Q&A] [answered]
For baseline (3.1), the question says that

```
Transform your model into a token classification model by imposing  softmax  over the tokens instead of predefined classes. You will need to do this twice for each of start and end.
```
I am confused about the wording "predefined class". Do you mean we shouldn't be using three classes as suggested in the lecture? Does it mean that our labels should be start and end token indices? It feels rather awkward to define it that way.  

EDIT: Did you mean we shouldn't be doing sequence classification by this?

**Comment by seominjoon** [comment, marked as answer]
Question answering is not a sequence (text) classification because it is not being classified into pre-defined classes. Rather, you are classifying each token if it is start or not (and end or not).

There are two ways to do this. one is that you can use sigmoid to individually classify each token. The second is that you can use softmax among the tokens. You are being asked to use the latter.

**Reply by ggoggam** [reply]
I understand it now. Thank you!
